app/views/templates/script/package_action.erb

```diff
@@ -83,6 +83,18 @@ handle_zypp_res_codes () {
 }
 <% end -%>
 
+<% if package_manager == 'apt' && action == 'update' -%>
+get_upgradeable_packages () {
+  packages=""
+  for i in $1; do
+    if [ $(LANG=C dpkg-query -W -f='${db:Status-Status}' "$i") = "installed" ]; then
```

Comment:
if the package is not installed, `dpkg-query` will not return anything on stdout resulting in:

```
# dash
dpkg-query: no packages found matching dpkgaaa
[: =: unexpected operator
# bash
dpkg-query: no packages found matching dpkgaaa
[: =: unary operator expected
```

```suggestion
    if [ "$(LANG=C dpkg-query -W -f='${db:Status-Status}' $i)" = "installed" ]; then
```

that would work